index.js

```diff
@@ -999,8 +1004,23 @@
               }
             } else {
               okTypes = valueTypes(values);
-              if (isEmpty(okTypes) && !isEmpty(values)) {
-                throw orphanArgument(env, name, _value, index);
+              if (!isEmpty(values)) {
+                //  If there's at least one value, and none of the values is
+                //  a member of a type in the environment, and all the values
+                //  have the same type identifier, the values are members of
+                //  a "foreign" type.
+                //
+                //  Create a nullary type for the foreign type, for inclusion
```

Comment:
I intended this to be read as plain English rather than as a reference to `$.NullaryType`.